/**:
  ros__parameters:
    update_rate: 10.0
    # Restricciones de velocidad:     **Deben coincidir con config drone_control.yaml**
    horizontal_vel: 6.0
    vertical_vel: 5.0